xla/service/gpu/dynamic_slice_fusion_rewriter.cc

```diff
@@ -44,6 +44,7 @@ limitations under the License.
 #include "xla/service/gpu/gpu_constants.h"
 #include "xla/service/gpu/hlo_traversal.h"
 #include "xla/service/gpu/ir_emission_utils.h"
+#include "xla/service/pattern_matcher.h"
```

Comment:
You will also need to add a corresponding dependency to the BUILD file.

Reply:
Added it.